lib/python/view.h

```diff
@@ -13,6 +13,17 @@ template <class T> class items_view {
   auto size() const noexcept { return m_obj->size(); }
   auto begin() const { return m_obj->items_begin(); }
   auto end() const { return m_obj->items_end(); }
+  auto tostring() const {
+    std::string out = "Items:";
+    if (this->size() == 0)
+      return out;
+    auto it = this->begin();
+    out += "\n" + it->first + ": " + to_string(it->second);
+    ++it;
+    for (; it != this->end(); ++it)
+      out += "\n" + it->first + ": " + to_string(it->second);
+    return out;
+  }
```

Comment:
Can't we re-use the code from the data array or dataset string generation )(potentially improved)? This feels like a lot of potential duplication.